src/JoyOI.Monitor/Models/Chart.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace JoyOI.Monitor.Models
+{
+    public class Chart
+    {
+        public string Title { get; set; }
+        public string Type { get; set; }
+        public ChartData Data { get; set; }
+        public Object Options { get; set; }
+    }
+
+    public class ChartData {
+        public List<string> Labels { get; set; }
```

Comment:
Please use IEnumerable instead of List, make sure don't use implementation type in class definition.